framework/helpers/BaseJson.php

```diff
@@ -90,6 +90,8 @@ public static function decode($json, $asArray = true)
     {
         if (is_array($json)) {
             throw new InvalidParamException('Invalid JSON data.');
+        } elseif ((string) $json === '') {
```

Comment:
It's better to check it explicitly with comparing to both null and empty string.